src/app/components/Breadcrumbs/Breadcrumbs.jsx

```diff
@@ -37,7 +37,7 @@ const Breadcrumbs = ({ query, type, title, url }) => {
           (<li>
             <Link
               title={`Make a new search for ${query}`}
-              to={`/search?q=${query}`}
```

Comment:
It does work but the `q=` is needed here because in the `ItemPage` component, `searchURL` can include the search keyword which is used with `q=`.